src/views/BusinessCase/AlternativeSolution/index.tsx

```diff
@@ -0,0 +1,230 @@
+import React from 'react';
+import { Field, FormikProps } from 'formik';
+import Label from 'components/shared/Label';
+import HelpText from 'components/shared/HelpText';
+import TextField from 'components/shared/TextField';
+import TextAreaField from 'components/shared/TextAreaField';
+import FieldGroup from 'components/shared/FieldGroup';
+import FieldErrorMsg from 'components/shared/FieldErrorMsg';
+import Button from 'components/shared/Button';
+import EstimatedLifecycleCost from 'components/EstimatedLifecycleCost';
+import { defaultProposedSolution } from 'views/BusinessCase';
+import {
+  BusinessCaseModel,
+  ProposedBusinessCaseSolution
```

Comment:
I chose the name `ProposedBusinessCaseSolution` because it captures the Preferred Solution as well as the proposed alternatives.

I think `BusinessCaseSolution` can also capture the "as is" solution, which we don't want to capture.